bagit.py

```diff
@@ -579,9 +585,50 @@ def files_to_be_fetched(self):
         local filename
         """
 
-        for url, file_size, filename in self.fetch_entries():
+        for _, _, filename in self.fetch_entries():
             yield filename
 
+    def fetch(self, force=False):
```

Comment:
I'm wondering whether we should support `force()` or simply have the user handle file operations outside of scope. The main thing which that does is open up the need to think about potential security concerns so perhaps that should just be confirming that we correctly validate file paths not being outside of the bag scope.